package.json

```diff
@@ -17,10 +17,10 @@
     "0ungit-credentials-helper": "./bin/credentials-helper"
   },
   "dependencies": {
-    "async": "~1.5.2",
-    "bluebird": "~3.2.2",
-    "blueimp-md5": "~2.1.0",
-    "body-parser": "~1.14.2",
+    "async": "~2.0.0-rc.2",
```

Comment:
This bump breaks the directory autocomplete since [`async.filter` now wants an error object as first argument](https://github.com/caolan/async/blob/master/CHANGELOG.md#breaking-changes).

Will submit a PR shortly.